entity-framework/efcore-and-ef6/porting/index.md

```diff
@@ -8,7 +8,7 @@ uid: efcore-and-ef6/porting/index
 # Porting from EF6 to EF Core
 
 Because of the fundamental changes in EF Core we do not recommend attempting to move an EF6 application to EF Core unless you have a compelling reason to make the change.
-You should view the move from EF6 to EF Core as a port rather than an upgrade.
+You should view the move from [EF6](https://docs.microsoft.com/en-us/ef/ef6/) to [EF Core](https://docs.microsoft.com/en-us/ef/core/) as a port rather than an upgrade.
```

Comment:
You should remove the language part from the link

Reply:
HI, I didn't get it. Pls help.

Thanks,
Ashish